src/styles/blocks/container.scss

```diff
@@ -0,0 +1,27 @@
+.container {  
+  margin: 0 auto;
+  padding: 36px 120px;
```

Comment:
On my screen block Header has hight more than 100vh. I'm not sure, but maybe you have to set padding-bottom on all Header, not for container.